editor/src/clj/editor/field_expression.clj

```diff
@@ -45,7 +45,7 @@
 
 (defn to-double [s]
  (try
-   (evaluate-expression #(Double/parseDouble %) 0.01 s)
+   (evaluate-expression #(Double/parseDouble %) math/precision-general s)
```

Comment:
Use the same rounding as everything else for the results of expressions typed into numeric fields.